eclipse-theia/theia-blueprint
# Theia 1.25.0 Update and Getting Started Update

#### What it does
* Update Theia to 1.25 and upgrade dependencies
* Add Link to Theia support page on welcome page

#### How to test
* Build and check functionality
* click support link on welcome page

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
